src/app/components/Text/index.stories.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import Text from './index';
+
+const paragraphBlock = (blockId, text) => ({
+  blockId,
+  model: {
+    text,
+  },
+});
+
+const props = {
+  blocks: [
+    paragraphBlock('01', 'This is a text block.'),
```

Comment:
Could you please update the copy here to be 'this is a paragraph', etc. instead of a text block? 
It'll make the story in Storybook more accurately map to the actual underlying data from the CMS. Here there is a single text block with three paragraphs. 

It's a little unfortunate that the text attributes on paragraphs are different to text blocks. So we should try time clarify things as much as possible on our side.